### ISSUE TITLE
[BUG] Native macOS Universal AutoFill leaving the 'password' field blank on some websites

### ISSUE THREAD

**Issue by seakrebel** [strongbox-password-safe/Strongbox#815]
### Preliminaries 
I have ensured that:

- [x] I am running the latest version of Strongbox on the App Store by searching for Strongbox and clicking into it to see the Update button (or not)
- [x] I have performed a full restart of my device no matter how annoying that is

### Version
 - OS: MacOS Sequoia Version 15.0.1 (Build 24A348)
 - Strongbox: 1.60.23 (5648)-P

### Bug Description
When using macOS Universal AutoFill to input credentials from Strongbox, the 'password' field is left blank on certain websites, even though the username fills correctly. This issue is inconsistent, as AutoFill works perfectly on some sites but fails on others. Importantly, these websites where Universal AutoFill fails to fill the **password** from Strongbox are compatible with the Apple Passwords, indicating that the HTML form structure is not the issue.

### Steps to Reproduce

1. **Create an entry** in Strongbox with a username and password for a test site (e.g., `https://jack-wolfskin.de`).
2. **Enable AutoFill for Strongbox**:
   - On macOS, go to **System Settings > AutoFill & Passwords** and enable AutoFill for Strongbox only.
   - In Strongbox, go to **Database > AutoFill Settings** and ensure **Enable AutoFill for this Database** is enabled. (I also checked everything inside Advanced Settings just in case)
3. **Enable AutoFill in Safari**:
   - In Safari, go to **Settings > AutoFill** and confirm that 'User names and passwords' is enabled.
4. **Test AutoFill**:
   - Open Safari and navigate to `https://jack-wolfskin.de`.
   - Attempt to use AutoFill for the username and password fields.

**Expected Result:** Both username and password fields are populated by AutoFill.

**Actual Result:** Only the username is filled, while the password field remains blank.

### Screenshots & Recordings

https://github.com/user-attachments/assets/0a145636-666c-4bb7-8203-0347b3b7b484

**Comment by strongbox-mark** [comment]
Thanks for the report, that is odd indeed and I can see the same issue. Do you have a list of other affected sites? 

If Apple's own Passwords app is able to fill then Strongbox should be able to do the same...

**Comment by strongbox-mark** [comment]
Hi u/kayg04, can you confirm if this also happens for Apple's own Passwords app?

**Comment by kayg04** [comment]
No, the sad thing is I have to resort to the Passwords app then because that works perfectly. I can record a video if you’d like.

**Comment by strongbox-mark** [comment]
Yes please, if you could include the URL and version of Safari/Strongbox etc and send them to support@strongboxsafe.com, we'll take a look.

**Comment by kayg04** [comment]
> Yes please, if you could include the URL and version of Safari/Strongbox etc and send them to [support@strongboxsafe.com](mailto:support@strongboxsafe.com), we'll take a look.

URL: https://www.figma.com/password/recover

Passwords.app:

![Image](https://github.com/user-attachments/assets/1ea89b13-c507-4d28-9ffd-3c9cbdca1667)

Strongbox:

![Image](https://github.com/user-attachments/assets/0f22e721-101e-4a43-87d9-4dbfaa4072fb)

https://github.com/user-attachments/assets/5d828bdf-db78-40a2-8ea3-560c89740b45

**Comment by strongbox-mark** [comment]
Hi u/kayg04, Strongbox appears to work for me on both the Figma recover page and the online SBI login page using the same version of Safari. I'm having trouble viewing or downloading your screen recording I'm afraid. At this point, I'd recommend you perform a full factory reset of Strongbox and remove all copies and reinstall. I'd actually recommend the following:

https://strongbox.reamaze.com/kb/troubleshooting-and-errors/how-do-i-completely-remove-slash-nuke-all-strongbox-settings-on-macos

Perform both actions, the first factory reset and then use the Terminal to clear any settings. Make sure all versions of Strongbox are removed from your device, restart your Mac, and reinstall. Let us know how that goes.
